Formula/wiredtiger.rb

```diff
@@ -1,8 +1,8 @@
 class Wiredtiger < Formula
   desc "High performance NoSQL extensible platform for data management"
   homepage "http://www.wiredtiger.com"
-  url "https://github.com/wiredtiger/wiredtiger/releases/download/3.2.0/wiredtiger-3.2.0.tar.bz2"
-  sha256 "c812d34ac542fdd2f5dc16e2f47ebc1eba09487f45e34fbae5a052a668931968"
+  url "https://github.com/wiredtiger/wiredtiger/releases/download/10.0.0/wiredtiger-10.0.0.tar.bz2"
+  sha256 "4830107ac744c0459ef99697652aa3e655c2122005a469a49d221e692fb834a5"
 
```

Comment:
Upstream License file states
> This program is free software: you can redistribute it and/or modify it underthe terms of either version 2 or version 3 of the GNU General Public Licenseas published by the Free Software Foundation.

So I think the license should be 
```suggestion
license "GPL-2.0-or-later"
```

Reply:
Doesn't this mean
```
license any_of: ["GPL-2.0-only", "GPL-3.0-only"]
```
? Since they've technically excluded a hypothetical future GPL-4.0.

Reply:
I guess that is probably more accurate
https://source.wiredtiger.com/10.0.0/license.html